app/views/dashboard/get.html.erb

```diff
@@ -15,3 +15,12 @@
     <% end %>
   </table>
 <% end %>
+<br>
+<br>
+New Semester
+<form method="POST">
```

Comment:
since we know all possible classes, semesters, offerings, etc. Sam can add maybe we can use drop down lists such that all sam should do is just pick the thing he wants from drop down menu and that's it.